multimedia/M2VDownsizer/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                M2VDownsizer
+version             1.1
+revision            0
+
+categories          multimedia
+platforms           darwin
```

Comment:
This is the default.
```suggestion
```

Reply:
Yes.  Since I included `platforms` on past ports, I included it here.  If MacPorts no longer wants `platforms  darwin`, I have no problem with dropping it.